install/kubernetes/helm/istio/charts/kiali/templates/configmap.yaml

```diff
@@ -11,12 +11,17 @@ metadata:
 data:
   config.yaml: |
     istio_namespace: {{ .Release.Namespace }}
+    auth:
```

Comment:
are the auth strategies specified in some document somewhere?  IF so, can you add the link so we can document the change in the website and 1.2 if this is backported?

Reply:
This actually is backward compatible as-is - Kiali is going to behave the same as before (so for users installing via helm or upgrading, the behavior of Kiali doesn't change - the user will need to log into Kiali using credentials stored in the secret). The different auth strategies are documented here: https://www.kiali.io/documentation/getting-started/#_login_options - I can add that link to this configmap.yaml

Note: for Kubernetes users, the only ones applicable are "anonymous" and "login" (OpenShift has an additional one - "openshift" which utilizes the OpenShift OAuth client capability found only in OpenShift).